src/python.cpp

```diff
@@ -1,13 +1,22 @@
 #include "SmurfTransmitter.h"
 #include "SmurfBuilder.h"
 
+#define PY_ARRAY_UNIQUE_SYMBOL Py_Array_API_SO3G
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#include <numpy/arrayobject.h>
```

Comment:
I am uneasy about all this numpy stuff, borrowing the so3g symbol space.  Do you still need any of this, given that you're not creating numpy arrays in this version?  The one thing you might need is to add bp::import("so3g") in the module init, to makes sure that so3g has initialized its numpy structures before you call the G3SuperTimestream stuff that counts on numpy API.